### PR TITLE
Added Dockerfile.full

### PR DESCRIPTION
**Pull request by SimoneLazzaris**
This dockerfile includes both immuadmin and immuclient in a single container.

### CODE REVIEW

File: build/Dockerfile.full [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    IMMUDB_DEVMODE="true" \
+    IMMUDB_MAINTENANCE="false" \
+    IMMUDB_ADMIN_PASSWORD="immudb" \
+    IMMUDB_PGSQL_SERVER="true" \
+    IMMUADMIN_TOKENFILE="/var/lib/immudb/admin_token" \
+    USER=immu \
+    HOME="${IMMUDB_HOME}"
+
+COPY --from=build /src/immudb /usr/sbin/immudb
+COPY --from=build /src/immuadmin /usr/local/bin/immuadmin
```
Comment (tomekkolo) [inline build/Dockerfile.full:78]: I think you are missing immuclient here

File: webconsole/statik.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
...
...xfc\xe0\xff\xdbG\xfe\xf0/t\xe7\xc6\xed\x1fZ\xcc\xce!\x941\xe3\xf0\xddX\xe7\xab\x8e\xf0\xd0\xee\x80\xe56\x16^\xb3 \x8f3\xf8\xe9Q,\xa2\x04\x07\xf3\xd6\xcc\xad\x8b\xef\xeb\x14\x0b\x1e\xbe\xe5\xa7\x07\xa9\xcf\xa3\x8e\xd3r\x8f\xdf\\\x96O7\x0b\xa3\xed\x85\xd0\xf1\x01c\xfd\xd1.\xce\xfa\xf0WO\xc1\xbb\x98\xc77\xd7\x1d\x95T\x81\xaf\xae\x1f\x1ek\xd4\xac\xb7\x99\xee\xd1\xc1\x17`\x87S\xb8Zq\n+\xf9y\xb2o\x12\xf1\xdc\x99&/O&;\xdf\xd4\x99&^\x1c\xb9qsH(\xc3\xbb\xa6\x86\xd4\xb2\x80{\xdc\x08\xfd2\x11b\xb2\x1f+\xa6\x9c\xf4\xaf\x8f\xe9\xa4\xa4N\xdf]\xcb\x9f\xca\x05\x17\xffy\xe4>~\xebI\x00\xfa\x9d\xca\xca\x1elr\x1df\xb7-!\xf9\xd2q\xc7\x12\x0b\x0b\x0bKCU\x07Q\xafl\x13\xf1?\x01\x00\x00\xff\xffPK\x07\x08{\x86\x02V\x89p\x00\x00\x99p\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\xe4HES\xaf\x13j\x89\x12\x02\x00\x00\x14\x04\x00\x00\x12\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xb4\x81\x00\x00\x00\x00missing/index.htmlUT\x05\x00\x01\xbd\x15\\aPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\xabe\xdeR{\x86\x02V\x89p\x00\x00\x99p\x00\x00\x12\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xb4\x81[\x02\x00\x00missing/mascot.pngUT\x05\x00\x01bg\xdc`PK\x05\x06\x00\x00\x00\x00\x02\x00\x02\x00\x92\x00\x00\x00-s\x00\x00\x00\x00"
```
Comment (arriqaaq) [inline webconsole/statik.go:13]: Hello @SimoneLazzaris !

This statik file is not committed usually. Possible to exclude it?

File: build/Dockerfile.full [hunk 3]
```diff
@@ -0,0 +1,50 @@
+FROM golang:1.18 as build
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download -x
+COPY . .
+RUN rm -rf /src/webconsole/dist
```
Comment (arriqaaq) [inline build/Dockerfile.full:6]: Can this be added to .dockerignore? Or am I missing something in understanding this?

Comment (arriqaaq) [reply build/Dockerfile.full:6]: I see the same approach is used in based dockerfile

File: build/Dockerfile.full (hunk 1 above)
Comment (SimoneLazzaris) [reply build/Dockerfile.full]: I worked on previous target... this one is removed

File: webconsole/statik.go (hunk 2 above)
Comment (SimoneLazzaris) [reply webconsole/statik.go]: Excluded in commit 2071adbe

File: build/Dockerfile.full (hunk 3 above)
Comment (SimoneLazzaris) [reply build/Dockerfile.full:6]: I don't even have a `/webconsole/dist` directory on my machine... I don't know why this is done (I don't know how webconsole is built) but what you say make totally sense.

Comment (SimoneLazzaris) [reply build/Dockerfile.full:6]: Fixed in commit 5ca03f68
I guess we can fix also other dockerfiles

Comment (arriqaaq) [reply build/Dockerfile.full:6]: I agree with you, thanks for fixing this Simone, I'll try to add it in other dockerfiles too

**Review by arriqaaq — APPROVED** [review]

**Pull request merged**